src/pages/home/component/carousel/carousel-data.js

```diff
@@ -0,0 +1,7 @@
+export const CarouselData = [
```

Comment:
if it is possible you can shift the data to the backend.....it would be easier to change the images in future